pom.xml

```diff
@@ -27,6 +31,25 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+
+		<dependency>
```

Comment:
Should we keep this dependency?

Reply:
I don't understand, which dependency you mean. If you mean spring-boot-starter-test, so this dependency is important for test's work. If you mean tomcat-embed-jasper, so this dependency is important as well, because app can't be run without it. Dependencies  tomcat-embed-jasper and spring-boot-starter-tomcat must be used together